### PR TITLE
Added Cloudfront Response Header changes per Nov 4 updates.

### PR DESCRIPTION
**Pull request by glitchindustries**
Adding Changes per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudfront-responseheaderspolicy.html

### CODE REVIEW

**Review by ntwaddell — APPROVED** [review]

File: troposphere/cloudfront.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
     cloudfront_origin_request_query_string_behavior,
     cloudfront_restriction_type,
     cloudfront_viewer_protocol_policy,
+    cloudfront_access_control_allow_methods,
```
Comment (markpeek) [inline troposphere/cloudfront.py:19]: These should be alphabetized in the list.

File: troposphere/cloudfront.py [hunk 2]
```diff
@@ -401,6 +404,130 @@ class OriginRequestPolicy(AWSObject):
     }
 
 
+class AccessControlAllowHeaders(AWSProperty):
+    props = {
+        "Items": [(str, True)]
```
Comment (markpeek) [inline troposphere/cloudfront.py:409]: The form `[(str, True)]` should be `([str], True)`. In other words, the tuple is (property_type, required_bool) and the type is an array of str `[str]`.

File: troposphere/cloudfront.py [hunk 3]
```diff
@@ -401,6 +404,130 @@ class OriginRequestPolicy(AWSObject):
...
+    }
+
+
+class AccessControlAllowMethods(AWSProperty):
+    props = {
+        "Items": [(cloudfront_access_control_allow_methods, True)]
```
Comment (markpeek) [inline troposphere/cloudfront.py:415]: Same change needed.

File: troposphere/cloudfront.py [hunk 4]
```diff
@@ -401,6 +404,130 @@ class OriginRequestPolicy(AWSObject):
...
+    }
+
+
+class AccessControlAllowOrigins(AWSProperty):
+    props = {
+        "Items": [(str, True)]
```
Comment (markpeek) [inline troposphere/cloudfront.py:421]: Same change needed.

File: troposphere/cloudfront.py [hunk 5]
```diff
@@ -401,6 +404,130 @@ class OriginRequestPolicy(AWSObject):
...
+    }
+
+
+class AccessControlExposeHeaders(AWSProperty):
+    props = {
+        "Items": [(str, True)]
```
Comment (markpeek) [inline troposphere/cloudfront.py:427]: Same change needed.

**Review by markpeek — CHANGES_REQUESTED** [review]
Thank you for making this change. There needs to be a couple of easy tweaks done. Also, it would be good to run `make fix-black fix-isort test lint` just to verify the change before pushing it again. Thanks!

**Review by markpeek — APPROVED** [review]
Looks great!

**Comment by ntwaddell** [comment]
Woohoo!

**Comment by markpeek** [comment]
Thanks!

**Pull request merged**